Sources/Shared/Request+Authentication.swift

```diff
@@ -36,7 +36,8 @@ private let UsernameKey = "username"
 private let PasswordKey = "password"
```

Comment:
What do you think of moving all of these constants to the bottom of this file so they are out of the way?

Reply:
I'm down - I moved only the private constants to the bottom and left the public typealias on top if that sounds good